spec/app/ics-028-cross-chain-validation/methods.md

```diff
@@ -440,10 +445,12 @@ function CreateConsumerClient(p: SpawnConsumerChainProposal) {
 - **Precondition** 
   - `currentTimestamp() > p.spawnTime`.
 - **Postcondition** 
-  - A client state is created with `chainId = p.chainId` and `unbondingPeriod` set to `ComputeConsumerUnbondingPeriod(stakingKeeper.UnbondingTime())`.
-  - A consensus state is created with `validatorSet` set to the validator set the provider chain own consensus state at current height.
-  - A client of the consumer chain is created and the client ID is added to `chainToClient`.
-  - `lockUnbondingOnTimeout[p.chainId]` is set to `p.lockUnbondingOnTimeout`.
+  - If a client for `p.chainId` already exists, the state is not changed.
```

Comment:
Why would this ever be the case?

Reply:
It's a sanity check. Let's say that there is already a consumer chain `C1` with `chainId`, with a client, connection and channel established. If a proposal passes to create another consumer chain `C2` with the same `chainId`, then this can break the CCV channel to `C1`.

Reply:
Most likely, such a proposal should not pass, but I'd prefer to not have to trust validators that they're checking the `chainId`s.

Reply:
Ok sounds good :+1:, just wanted to make sure i wasn't missing something in my understanding